### DISCUSSION TITLE
Issues in setting up and using the package

### DISCUSSION THREAD

**Discussion by skb1129** [skb1129/react-native-change-icon#31] [Q&A]
Drop any problems you face in setting up and using the package.

**Comment by SalmanMalyk** [comment]
I'm getting this error:  _reactNative.NativeModules.ChangeIcon.getIcon is not a function.

**Reply by skb1129** [reply]
@SalmanMalyk which React-Native version are you using? Which device are you using?
This is only possible when the package linking is not done.

**Comment by SalmanMalyk** [comment]
0.67.X.
Can you tell me how to link this?
I'm new to React Native.

**Reply by skb1129** [reply]
Android should automatically link when the package is installed using npm
For iOS you will need to re-install cocoapod dependencies for which you can run run `npx pod-install ios`

**Reply by SalmanMalyk** [reply]
I'm trying to make it for Android but it is not working. 
BTW i am using yarn for installing package.
